rust.md

```diff
@@ -0,0 +1,39 @@
+# Rust
+A checklist for people using Rust to develop Firefox services, to be used in addition to the [common checklist](README.md).
+
+* [ ] Use https://github.com/RustSec/cargo-audit to check for dependency security vulnerabilities
+* [ ] Use https://github.com/rust-lang/rust-clippy for linting (easier for new projects)
+* [ ] Use https://github.com/mozilla-services/slog-mozlog-json for service logging
+* [ ] Use a Cargo.lock file (which is also required for cargo-audit)

```

Comment:
We should clarify whether we're talking about libraries or binaries: https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

Reply:
Agreed - have updated the checklist to state that.